_projects/authenticated_3rd_party_invites.md

```diff
@@ -0,0 +1,104 @@
+---
+name: 3rd Party Authorised Room Membership
+desc: Invite and kick users in a room based on some external authorisation. 
+requirements:
+ - Understanding of the matrix Client-Server API
+ - Familiarity with Javascript & React
+ - Familiarity with a programming language with a good matrix SDK (for the
+   server side component).
+difficulty: medium
+issues:
+mentors:
+ - Cadair
+ - Half-Shot
+initiatives:
+ - GSoC
+tags:
```

Comment:
Some of the pure tech requirements from `requirements` should probably be moved here. I'd leave "Understanding of the matrix Client-Server API" under `requirements` though